torch/csrc/autograd/engine.h

```diff
@@ -102,6 +102,12 @@ struct GraphTask {
   // an exception as soon as the autograd engine receives an exception.
   bool exit_on_error_;
 
+  // cpu ready queue asssociated with each graph task, since each `backward()`
+  // or `grad()` have its own GraphTask, we memorize the CPU thread's ReadyQueue
+  // in GraphTask to support cross device training (i.e. GPU, XLA to CPU via
+  // variable.cpu(), etc.)
```

Comment:
This comment is very vague. How exactly does it help with cross device training?